apps/e2e/indicators-storybook/src/app/text-highlight/text-highlight.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-text-highlight',
+  templateUrl: './text-highlight.component.html',
+  styleUrls: ['./text-highlight.component.scss'],
+})
+export class TextHighlightComponent {
+  public highlightSentence = " Doesn't it look great.";
```

Comment:
Very minor - I wouldn't have left the comment if I hadn't already left another - but I would personally not add the extra space before the sentence so that the highlighting begins at the beginning of the sentence.

Reply:
I added it mainly because, i could see a situation in the future where we add "ignore trailing whitespace on either side" for text highlight and we want that captured in a visual test.